locales/templates/app.pot

```diff
@@ -603,12 +607,14 @@ msgstr ""
 
 #. String displayed with an illustration to introduce a new feature
 msgctxt "onboarding_ocr_title_more"
-msgid "Do more with screenshots"
+msgid "Take Action on Text"
```

Comment:
The string is changing entirely, so better update the string entity here so localizers will see that they have to translate it again

Reply:
Thank you for noticing it. Based on experience, the updated strings all got localizers' attention. Also, changing the string entity might cause difficulties to our string maintenance to engineers. So I would suggest keeping the string entity. Thanks.

Reply:
@benjamin-cheng I think this string was overlooked because there were two strings concerned by the "fuzzy" statement I made :)

Reply:
@Delphine "onboarding_ocr_title_more" has been marked as "fuzzy" at https://github.com/mozilla-l10n/scryer-android-l10n/pull/10/files#diff-2270efcf5b50f573604e9f8f75128ba2R595